string file_name
string format_name
opencv_apps/RotatedRect[] parts
---
string[] names